dist/s_string.ok

```diff
@@ -174,6 +174,17 @@ Hendrik
 IAA
 IC
 IEC
+IIII
+IIIIu
+IIIUUI
+IIIUu
+IIIr
+IIIrr
+IIIru
+IIIu
+IIIuu
+IIIuuI
+IIQQQQQQQ
```

Comment:
Do we need these anymore?

Reply:
Yes, as the new function comment is referring to those string literals, they must be mentioned here.